.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_COUNTRY_VARIANT="cs"
```

Comment:
Actually it might be fine to include as a default, just check it does not break the Vercel build. Maybe name it `.env.development` ? per https://nextjs.org/docs/basic-features/environment-variables

Reply:
Sounds good. I think that we should also mention somewhere that even though `.env.development` exists, either `.env.production`/`.env.local` should be created as well with the same variables in order to  `npm run build` and `npm start`. Can we place it in `.env.example` ? or in README? Or there is another option, just to define it in Vercel settings as development variable and pull with `vercel env pull .env.local` but user should have vercel CLI installed which is probably overkill for one variable. What do you think is the best option?

Reply:
No no. Functionally, Vercel provides the variable in the production environment and the developer provides the variable in the development environment by specifying a `.env.local` file which does not get tracked in git. 

The only purpose of `.env.development` is as a template for creating `env.local` by the developer.

Reply:
Fixed it